lib/utils/dataStructureUtils.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+function createMapFromObject(obj) {
+    const map = new Map();
+    for (const key in obj) {
+        map.set(key, obj[key]);
+    }
+    return map;
+}
+exports.createMapFromObject = createMapFromObject;
+function isArrayOfString(x) {
+    return (Array.isArray(x) &&
+        x.every(item => Boolean(item && typeof item === "string")));
+}
+exports.isArrayOfString = isArrayOfString;
+function isArrayOfObject(x) {
```

Comment:
I think you should add `s` to the end. `isArrayOfObjects`

Reply:
I assumed there was a reasoning behind these namings. Was there @bilgeyucel ?

Reply:
Nope, there wasn't. `isArrayofObjects` is grammatically correct naming👍